<?xml version="1.0"?>
<launch>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="turtle_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- ros_serial node -->
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/sensors/servo_arduino"/>
  </node>
 
 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="150" type="double"/>
  <param name="scale_angular" value="250" type="double"/>

  <node pkg="argo_base_control" type="argo_base_control_node" name="argo_base_controller"/>



</launch>
